### DISCUSSION TITLE
"Jekyll build failed" on a site with .nojekyll

### DISCUSSION THREAD

**Discussion by stur86** [community/community#22647] [Other Feature Feedback, Questions, & Ideas]
<p>I’ve just pushed a change to my repository and have received a message warning me that my Jekyll build has failed. The page is still online, but in Settings for the repository I still find an error message instead of the usual “your page is online” and the URL green message. However, I have put .nojekyll files both in the master and docs folders (I’m using docs for the page), so of course I wouldn’t expect the build to begin in the first place. Is this normal behaviour or some kind of bug? Is it an error on my part? This is the repository:</p>
<p><a href="https://github.com/CCP-NC/soprano" rel="nofollow noopener">https://github.com/CCP-NC/soprano</a></p>
<p>Thank you very much!</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/stur86">@stur86</a>,</p>
<p>Thanks for being here! I want to confirm that I see your site is up, but if you’re still seeing this error or other strangeness, I’d recommend you write in to <a href="https://github.com/contact" rel="nofollow noopener">private support</a>. They’ll be able to dive into this a bit deeper and help discover what the underlying issue might be.</p>
<p>Cheers!</p>

**Comment by stur86** [comment]
<p>Thanks! I don’t see the error any more now. No idea what the deal was. Problem closed for me, sorry for the false alarm!</p>

**Comment by nadiajoyce** [comment]
<p>Oh, I’m so glad all is well! Thanks for the update and letting us know!</p>
